src/WebJobs.Script.WebHost/Metrics/FlexConsumptionMetricsPublisher.cs

```diff
@@ -112,10 +113,10 @@ internal async Task OnPublishMetrics(DateTime now)
 
                 // we've been accumulating function activity for the entire period
                 // publish this activity and reset
-                Metrics metrics = null;
+                PublishMetrics metrics = null;
                 lock (_lock)
                 {
-                    metrics = new Metrics
+                    metrics = new PublishMetrics
```

Comment:
Please revert back to having the existing Metrics class scoped to just this class